misk-aws-dynamodb-testing/src/main/kotlin/misk/aws/dynamodb/testing/DockerDynamoDbModule.kt

```diff
@@ -11,14 +12,18 @@ import kotlin.reflect.KClass
  * Spins up a docker container for testing. It clears the table content before each test starts.
  */
 class DockerDynamoDbModule(
-  private val tables: List<KClass<*>>
+  private val tables: List<DynamoDbTable>
 ) : KAbstractModule() {
 
-  constructor(vararg tables: KClass<*>) : this(tables.toList())
+  constructor(vararg tables: DynamoDbTable) : this(tables.toList())
+  constructor(vararg tables: KClass<*>) :
+    this(
+        tables.toList().map { DynamoDbTable(it) }
```

Comment:
Would `tables.toList().map { DynamoDbTable(it) }` work?

Reply:
it does. updating